Lib/ast.py

```diff
@@ -56,14 +56,14 @@ def _convert(node):
         elif isinstance(node, Num):
             return node.n
         elif isinstance(node, Tuple):
-            return tuple(map(_convert, node.elts))
+            return tuple(_convert(v) for v in node.elts)
```

Comment:
In all of these cases, I prefer the original for clarity and inter-line parallelism